public/index.php

```diff
@@ -21,7 +21,7 @@
 |
 */
 
-require __DIR__.'/../vendor/autoload.php';
+require __DIR__.'\..\vendor\autoload.php';
```

Comment:
Never seen this done before. Assuming this makes it work on windows, but doesn't it then break on Mac?

There must be another way - this file should stay as the default.